spec/graphql/fragment_cache/object_helpers_spec.rb

```diff
@@ -777,107 +777,6 @@ def post(id:, expires_in: nil)
     end
   end
 
-  describe "conditional caching" do
-    let(:schema) do
-      field_resolver = resolver
-
-      build_schema do
-        query(
-          Class.new(Types::Query) {
-            field :post, Types::Post, null: true do
-              argument :id, GraphQL::Types::ID, required: true
-            end
-
-            define_method(:post, &field_resolver)
-          }
-        )
-      end
-    end
-
-    let(:id) { 1 }
-    let(:variables) { {id: id} }
-
-    let(:query) do
-      <<~GQL
-        query getPost($id: ID!) {
-          post(id: $id) {
-            id
-            title
-          }
-        }
-      GQL
-    end
-
-    let!(:post) { Post.create(id: 1, title: "object test") }
-
-    before do
-      # warmup cache
-      execute_query
-      # make object dirty
-      post.title = "new object test"
-    end
-
-    context "when :if is true" do
-      let(:resolver) do
-        ->(id:) do
-          cache_fragment(if: true) { Post.find(id) }
```

Comment:
Should you keep those tests to ensure that the `cache_fragment(if: ...)` syntax continues to work in the future? Or do you think it is overkill.

Reply:
Thanks! Yeah, I'll bring them back.

(I was going to deprecate `if` and `unless` for `#cache_fragment` altogether, but since we're on v1, it would require the whole deprecated/whatever cycle—not this time 🙂)